sklearn/tests/test_metaestimators.py

```diff
@@ -269,8 +267,8 @@ def test_meta_estimators_delegate_data_validation(estimator):
     else:
         y = rng.randint(3, size=n_samples)
 
-    X = _enforce_estimator_tags_x(estimator, X)
-    y = _enforce_estimator_tags_y(estimator, y)
+    X = _enforce_estimator_tags_x(estimator, X).tolist()
```

Comment:
Maybe a small comment to state that we want something more general as array-like